src/sql/azdata.proposed.d.ts

```diff
@@ -1504,18 +1512,18 @@ declare module 'azdata' {
 		/**
 		 * Edge properties to be shown in the tooltip.
 		 */
-		properties: QueryPlanGraphElementProperty[]
+		properties: ExecutionPlanGraphElementProperty[]
 	}
 
-	export interface QueryPlanGraphElementProperty {
+	export interface ExecutionPlanGraphElementProperty {
 		/**
 		 * Name of the property
 		 */
 		name: string;
 		/**
 		 * Formatted value for the property
 		 */
-		formattedValue: string;
+		value: string | ExecutionPlanGraphElementProperty[];
```

Comment:
Is the comment still accurate about the formatted value?

Reply:
yes, it matches the latest contracts in STS.